src/main/kotlin/no/nav/dagpenger/inntekt/db/InntektStore.kt

```diff
@@ -5,18 +5,32 @@ import java.time.LocalDate
 import java.time.LocalDateTime
 import java.time.ZoneOffset
 import java.time.ZonedDateTime
-import no.nav.dagpenger.inntekt.BehandlingsKey
 import no.nav.dagpenger.inntekt.inntektskomponenten.v1.InntektkomponentResponse
 
 interface InntektStore {
     fun getInntekt(inntektId: InntektId): StoredInntekt
-    fun getInntektId(request: BehandlingsKey): InntektId?
+    fun getInntektId(inntektparametre: Inntektparametre): InntektId?
     fun getBeregningsdato(inntektId: InntektId): LocalDate
-    fun insertInntekt(request: BehandlingsKey, inntekt: InntektkomponentResponse, manueltRedigert: ManueltRedigert? = null, created: ZonedDateTime = ZonedDateTime.now(ZoneOffset.UTC)): StoredInntekt
+    fun storeInntekt(command: StoreInntektCommand, created: ZonedDateTime = ZonedDateTime.now(ZoneOffset.UTC)): StoredInntekt
     fun getManueltRedigert(inntektId: InntektId): ManueltRedigert?
     fun markerInntektBrukt(inntektId: InntektId): Int
 }
 
+data class Inntektparametre(
```

Comment:
Ordboka sier `Paramtere` for flertall 🥺

Reply:
hø. I følge språkrådet https://www.sprakradet.no/svardatabase/sporsmal-og-svar/parameter-kjonn-og-boying/